pyproject.toml

```diff
@@ -198,3 +198,45 @@ typeCheckingMode = "standard"
 # that is not meant to be exported from that module. The default value for this
 # setting is "error".
 reportPrivateImportUsage = false
+
+[tool.ruff]
+# exclude = ["__init__.py"]
+line-length = 120
```

Comment:
As an initial general comment, and because that setting seems to be responsible for most of the line change. What I say below is **subjective** and is at viewing distance of ~70cm (desk distance) on 27" screens.

At `line-length = 120` I cannot have all of:

1. a side-by-side view (e.g., to compare PR changes with 2 tabs open at the same time with a vertical split),
2. in a big enough font to be readable,
3. with all the lines complete on the screen.

I can mostly have the three points above with `line-length = 100` (I am missing the 5 last characters of each line, but I can see them by reducing the horizontal size of the file tree on the left).

So my opinion is that 120 is too large, 100 is OK.